images/monitoring/Dockerfile

```diff
@@ -21,10 +21,29 @@ ENV TOOL_NAME monitoring
 # Temporarily add test-infra to the image to build custom tools
 ADD ./tools $TEMP_TOOLS_REPO_DIR
 
+# Add cloud mail go dependencies first before extracting the source
+RUN go get -u cloud.google.com/go
+RUN go get -u google.golang.org/genproto/googleapis/api/annotations
+RUN go get -u google.golang.org/genproto/googleapis/iam/v1
+RUN go get -u google.golang.org/genproto/protobuf/field_mask
+
+#TODO(yt3liu): Use the alpha released after the go library builds correctly with the latest release
+#RUN wget https://storage.googleapis.com/cloud-mail-client-libraries/cloudmail-v1alpha3-go.zip
+RUN wget https://storage.googleapis.com/knative-monitoring/cloudmail-v1alpha3-go.zip
+
+RUN apt-get install zip unzip
+RUN unzip cloudmail-v1alpha3-go.zip -d cloudmail-v1alpha3-go
+RUN ls
```

Comment:
why do we need to run `ls`

Reply:
oops. forgot to remove it after checking where the file is extracted. Thanks for pointing it out. Removed.